pkg/nodebootstrap/assets/bootstrap.al2.sh

```diff
@@ -27,6 +27,13 @@ AWS_SERVICES_DOMAIN="$(get_metadata services/domain)"
 
 source /etc/eksctl/kubelet.env # this can override MAX_PODS
 
+INSTANCE_LIFECYCLE=$(aws ec2 describe-instances --instance-ids  ${INSTANCE_ID}  --query 'Reservations[0].Instances[0].InstanceLifecycle' --output text)
```

Comment:
We are trying to avoid adding extra dependencies to the bootstrap scripts. Please use the `get_metadata` function with `instance-life-cycle` instead of the `aws` cli.